interface/portraits/portraits_pandaren.gfx

```diff
@@ -11,26 +11,35 @@ spriteTypes = {
 		effectFile = "gfx/FX/portrait.lua"
 		layer = { # GFX_TYPE:[d|p]INDEX:COLOR_LINK:DONT_REFRESH_IF_VALID:CULTURE_INDEX
 			"GFX_character_background:p0"
+			"GFX_pandaren_male_hair_behind:p1:y"
+			"GFX_pandaren_male_beard_behind:p4"
 			
 			"GFX_pandaren_male_base:p2"
+			"GFX_pandaren_male_neck:d0"
+			"GFX_pandaren_male_clothes:p3"
+
+			"GFX_pandaren_male_beard:p4"
+			"GFX_pandaren_male_hair:p1:y"
+			"GFX_pandaren_male_ear:d7" #brow uses ear spot in ruler designer
+			"GFX_pandaren_male_headgear:p5"
 			
 			"GFX_worgen_druid_form:p19"
 			"GFX_character_imprisoned:p6"
 			"GFX_being_overlay:p20"
 			"GFX_player_overlay:p11"
 		}
 
-		hair_color_index = 8
+		hair_color_index = 1
 		hair_color = { # dark, base, highlight
-			{ 0 0 0 } {  125 125 125 } { 255 255 255 }	# Default gray
+			{ 0 0 0 } {  75 75 75 } { 255 255 255 } # Black
 		}
 
 		eye_color_index = 9
 		eye_color = {
 			{ 0 0 0 } {  125 125 125 } { 255 255 255 }	# Default gray
 		}
 
-		headgear_that_hides_hair = { }
+		headgear_that_hides_hair = { 11 }
```

Comment:
There are 3 headgears that should hide hairs.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/91372696-612d1580-e825-11ea-8883-c1b7f7514c4c.png)

</details>